### DISCUSSION TITLE
Application login with username / password not working via cypress alone

### DISCUSSION THREAD

**Discussion by mrsenzy** [cypress-io/cypress#22511] [Questions and Help]
Hi - We have a internal application that uses basic login / password with sign in button and than navigates to home screen. Its very simple screen - the login validation works fine while checking from local browser, via selenium script, at browserstack etc .. but trying to do same in cypress it gives login attempt failed with valid credentials. Below is the 4 line of cypress script that must work but it display login attempt failed. I am using cypress 9.5.0. I am able to follow same steps for other internal app's and those works fine  -- Have anyone faced this issue - Please share me any recommended approach for this issue.   

    const url = 'https://';
    cy.visit(url)
    cy.get('#username').first().type('su'.toLowerCase())
    cy.get('#password').first().type('gw'.toLowerCase())      
    cy.get('#signIn').click()

Below is screenshot of the logs displayed during sign in ... for reference. 
![image](https://user-images.githubusercontent.com/75691269/175568822-b04618a8-b1ac-49d9-b606-72b349f8f067.png)

**Comment by jjhelguero** [comment]
It's not clear what is the source of the issue. Is it the login info incorrect at the time you click signIn? Is your app not accepting those credentials as a valid user?

**Reply by mrsenzy** [reply]
Login details are correct. I am using the same exact while I manually login and I also tried same login approach with selenium script, it works there successfully. Login is not successful only via Cypress.
